deploy/kubernetes/metricbeat/metricbeat-role.yaml

```diff
@@ -33,3 +33,9 @@ rules:
   - "/metrics"
   verbs:
   - get
+- apiGroups:
+    - coordination.k8s.io
+  resources:
+    - leases
+  verbs:
+    - '*'
```

Comment:
Should we also include the changes to disable the deployment and extending the daemonset on this PR?
I would include the changes on the deployment and the permissions needed on the same PR.

Reply:
Hmm, I would go with a plan like:
1. Merge leader election implementation as standalone PR  (maybe removing the above lines too)
2. Follow up PR with a completely new manifest which will only consists of the Daemonset approach.
3. Decide and remove the Deployment manifests.

In this we will split the implementation with the proposed way to run in different PRs and we will make the transition smoother. wdyt?

Reply:
Yeah, better to do the changes on manifests on a different PR :+1: